app/views/menus/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<div> <%= image_tag "atrium.png", :class => "bg" %> </div>
+
+<div class="center-box">
+  <li>CAFE FLORA</li>
+  <li class = "divider">––––</li>
```

Comment:
Small style suggestion: dont' put spaces around the `=` in HTML, but in ruby do put the spaces. It's weird that they  are different, but that's the convention.